### PR TITLE
Weekly legacy doco review for 2019-07-29 

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@dbelcham I've covered the doco part today. Will try to cover the samples tomorrow.

**Comment by SzymonPobiega** [comment]
@dbelcham done. Can you review?

File: nservicebus/logging/index.md [hunk 1]
```diff
@@ -15,50 +15,57 @@ related:
 
 ## Default Logging
 
-NServiceBus has some limited, and opinionated, logging built in.
-
-The default logging behavior is as follows:
-
-This is applicable to both self hosting and using the NServiceBus Host
+NServiceBus has a built-in logging mechanism that does not depend on any external libraries. While limited in terms of available log targets, this build-in mechanism is production-ready and offers defaults that are reasonable for most deployments. The built-in framework is available and used as default in all NServiceBus hosting modes (e.g. self-hosting or Windows Service). Regardless if built-in or a custom library is used under the hood, the NServiceBus logging abstractions can be used for [logging in the user code](/nservicebus/logging/usage.md). By default NServiceBus has three log targets configured:
```
[the remark this answers is not in the document]
Comment (dbelcham) [reply nservicebus/logging/index.md]: > Regardless if built-in or a custom library is used....

would be better as
> Regardless of if the built-in logging or a custom logging library is used....

File: nservicebus/logging/index.md [hunk 2]
```diff
@@ -15,50 +15,57 @@ related:
...
 
 
 ### Console
 
-All `Info` (and above) messages will be piped to the current console.
+All `Info` (and above) messages are written to the current console, if available in the hosting environment.
```
Comment (dbelcham) [inline nservicebus/logging/index.md]: > written to the current console, if available in the hosting environment

better as
> written to the current console if one is available in the hosting environment

File: nservicebus/logging/index.md [hunk 3]
```diff
@@ -15,50 +15,57 @@ related:
...
 
 Errors will be written with `ConsoleColor.Red`. Warnings will be written with `ConsoleColor.DarkYellow`. All other message will be written with `ConsoleColor.White`.
 
 
 ### Trace
 
-All `Warn` (and above) messages will be written to `Trace.WriteLine`.
+All `Warn` (and above) messages are written to `Trace.WriteLine` and thus can be forwarded to any [trace listener](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/trace-listeners).
```
Comment (dbelcham) [inline nservicebus/logging/index.md]: > ...and thus can be...

better as
> ...and therefore can be...

File: nservicebus/logging/index.md [hunk 4]
```diff
@@ -15,50 +15,57 @@ related:
...
+Each log entry is associated with a _level_ that describes how important and critical that entry is. The built-in levels are following (in order of increasing importance)
 
+ * Debug
+ * Info
+ * Warn
+ * Error
+ * Fatal
 
-## Logging Levels
+Configuring global threshold to a specific level as described above means that all messages below that level are discarded. For example setting the threshold value to `Warn` means that only `Warn`, `Error` and `Fatal` messages are written.
```
Comment (dbelcham) [inline nservicebus/logging/index.md]: > Configuring global threshold to a specific level as described above means...

better as
> Configuring the global threshold to one of the levels described above means...

File: nservicebus/logging/index.md [hunk 5]
```diff
@@ -15,50 +15,57 @@ related:
...
 
-The Logging level, or "Threshold", indicates the log levels that will be outputted. So for example a value of `Warn` would mean all `Warn`, `Error` and `Fatal` message would be outputted.
+The `LogManager` class is the entry point for the logging configuration. If needed, it allows using custom logging integrations (see below). It also allows customizing the default built-in logging. The `Use` generic method returns the `LoggingFactoryDefinition`-derived object that provides the customization APIs.
```
Comment (dbelcham) [inline nservicebus/logging/index.md]: > It also allows customizing the default built-in logging

better as
> It also allows customization of the default built-in logging

**Comment by SzymonPobiega** [comment]
@dbelcham I applied your suggestions. Merging when green.

**Comment by dbelcham** [comment]
Green

**Pull request merged**